actions/ci-sonarqube-go/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "ci-sonarqube-go",
+  "version": "0.0.1",
```

Comment:
please use `pnpm changeset` to create a new changeset for this action with a `minor` semvar.

this should be `0.0.0` as the `Version Packages` PR created by changeset will bump it to `0.1.0`

Reply:
okay, fixed it. Thanks!

Reply:
you will still need to run `pnpm changeset` to generate a changeset file for this action.

Reply:
Generated a changeset file with major bump, or should I make it a minor bump?

Reply:
just a minor is fine

Reply:
Made the change to minor, thanks!